packages/angular/projects/clr-angular/src/data/datagrid/datagrid-column.ts

```diff
@@ -403,7 +403,7 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, ClrDa
 
   // This property holds filter value temporarily while this.filter property is not yet registered
   // When this.filter is registered, this property value would be used update this.filter.value
-  private initFilterValue: string | [number, number];
+  private initFilterValue: string | [number, number] | any;
```

Comment:
Same here. Considering that we are copy-pasting this type in multiple files, would it be better to make this a type alias?